blog/_posts/2023-06-07-toolkit.md

```diff
@@ -0,0 +1,282 @@
+```
+This blog post outlines the motivation and design of the Toolkit.
+It illustrates how to use it, including setup instructions for tools such as Scala CLI, sbt, Mill, and Maven.
+Finally, a FAQ section contains answers to commonly asked questions.
+
+## Acknowledgements
+
+We would like to express our gratitude and appreciation to the authors and maintainers of [com-lihaoyi/os-lib](https://github.com/com-lihaoyi/os-lib), [com-lihaoyi/upickle](https://github.com/com-lihaoyi/upickle), [softwaremill/sttp](https://github.com/softwaremill/sttp) and [scalameta/munit](https://github.com/scalameta/munit).
+Their collaborative efforts and dedication have been crucial in the successful creation of the Toolkit.
+We are looking forward to continued collaboration in the future.
+
+## Why the Toolkit?
+
+The Toolkit project is a significant step in our ongoing effort to make Scala more accessible, lightweight, and fun to use.
+In particular, we aim at lowering the onboarding barrier for newcomers from different programming backgrounds.
+
+Scala’s library ecosystem offers multiple choices for doing almost anything.
+The ecosystem’s richness is a great strength of the language, but can also be confusing and intimidating.
+The Toolkit aims to offer default choices for coders who aren’t in a position to evaluate competing libraries and just want to get something done.
+
+## What is the Toolkit?
+
+The Toolkit gathers a number of pre-existing libraries of the ecosystem that focus on concrete, recurring software engineering problems, such as working with files and processes, parsing JSON, and sending HTTP requests.
+
+Its documentation can be found at [docs.scala-lang.org/toolkit](https://docs.scala-lang.org/toolkit/introduction.html).
+It comprises a collection of concise tutorials, including short code examples, that aim at being self-contained.
+They not only explain how to use each library individually, but also how to combine them in meaningful ways.
+
+The Toolkit aims at being approachable.
+It does not require extended knowledge of the more advanced features of Scala (higher-kinded types, macros, etc) nor of the more advanced concepts of the ecosystem (asynchronous, pure-functional, reactive, actor-based).
+
+The Toolkit favors libraries that provide a user-friendly getting-started experience.
+They enable the developer to explore the API with auto-completion and integrated documentation (scaladoc), and discover the functionalities as they write their code.
+Additionally, they prioritize clear and informative types, signatures and error messages, rather than abstraction and flexibility.
+
+## What can you do with the Toolkit?
+
+Here is an example of what you can do with the Toolkit:
+```scala
+//> using toolkit 0.2.0
+
+import sttp.client4.quick.*
+
+val json = ujson.Obj(
+  "location" -> "hometown",
+  "bio" -> "Scala programmer"
+)
+
+val token = sys.env("GITHUB_TOKEN")
+
+val response = quickRequest
+  .patch(uri"https://api.github.com/user")
```

Comment:
Not sure if it's relevant, but it might be interesting for readers to have a comment with a link to the API doc here? `https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#update-the-authenticated-user`